Lib/test/test_collections.py

```diff
@@ -963,7 +963,7 @@ class AnextOnly:
             async def __anext__(self):
                 raise StopAsyncIteration
         self.assertNotIsInstance(AnextOnly(), AsyncIterator)
-        self.validate_abstract_methods(AsyncIterator, '__anext__', '__aiter__')
+        self.validate_abstract_methods(AsyncIterator, '__anext__')
```

Comment:
Since `AsyncIterator.__aiter__` is not abstract, it is correct to remove it here.